### ISSUE TITLE
Authenticate with cookie

### ISSUE THREAD

**Issue by LuisHansen** [sachbalag/ibm-maximo-api#11]
Is there a way to authenticate, or get the current user, using the cookies previously set?

**Comment by sachbalag** [comment]
Yes you can ... Invoke the authenticate api and it will send you the cookie in your promise handler. Store the cookie in the session in your request. After that, whenever you need it, pass the cookie as the second parameter in the Maximo constructor.

Here is an example, the first route invokes authenticate and stores it and the second one uses it.

app.get('/authenticate', function(req, res)
{
      var maximo = new Maximo(options);
      maximo.authenticate()
            .then(function(setcookie)
            {
              jsondata = setcookie;
              req.session.authcookie = jsondata; // Set the cookie in the session so we can use it for future requests
              res.json(jsondata);
            })
            .fail(function (error)
            {
                  console.log('****** Error Code = '+error);
            });
});

// Returns a Resource from ResourceSet - Resource #3
app.get('/test_resource', function(req, res)
{
      var authcookie = req.session.authcookie;
      console.log("********* AuthCookie "+authcookie);
      var maximo = new Maximo(options,authcookie);
      maximo.resourceobject("MXWODETAIL")
        .select(["wonum","description","reportedby","location","status","assetnum.assetnum"])
        .where("status").equal('WAPPR')
        //.and("location").equal('POLE300')
        .pagesize(20)
        .fetch()
        .then(function(resourceset)
          {
              req.session.myresourceset = resourceset.thisResourceSet();
              var rsrc = resourceset.resource(3);
              jsondata = rsrc.JSON();
              res.json(jsondata);
          }
        )
        .fail(function (error)
        {
              console.log('****** Error Code = '+error);
        });
});

**Comment by LuisHansen** [comment]
Thanks for the quick response!
What I really wanted was the user id, or some login information.
I need to use the current user info, but there doesn't seem to be any way to actually get that information.
